proposals/p2265.md

```diff
@@ -0,0 +1,67 @@
+# Name of application entry point
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/2265)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Problem](#problem)
+-   [Background](#background)
+-   [Proposal](#proposal)
+-   [Rationale](#rationale)
+-   [Alternatives considered](#alternatives-considered)
+
+<!-- tocstop -->
+
+## Abstract
+
+This proposal evaluates some possible reasoning which may contribute to a
+decision about naming the application entry point.
```

Comment:
It's kind of confusing not to mention that this proposal actually _makes_ that decision.

Reply:
That's a wonderful point and something that slipped past me during a second pass.  I will get that changed shortly Thanks!